assets/sass/components/global/_googlesitekit-icon-wrapper.scss

```diff
@@ -0,0 +1,25 @@
+/**
+ * HelpMenuLink styles.
```

Comment:
I think this needs to be updated to be reflective of the `IconWrapper` name as opposed to `HelpMenuLink`.